arbitrum-docs/stylus/stylus-gentle-introduction.md

```diff
@@ -9,123 +9,34 @@ sidebar_position: 1
 
 # A gentle introduction: Stylus
 
-import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.mdx';
-
-<PublicPreviewBannerPartial />
-
-This introduction is for developers who want to build on <a data-quicklook-from="arbitrum">Arbitrum</a> using popular programming languages like Rust. This capability is made possible by <a data-quicklook-from="stylus">Stylus</a>, a new way to write EVM-compatible smart contracts using your favorite programming languages.
-
 ### In a nutshell:
 
-- Stylus lets you write smart contracts in programming languages that compile to WASM, such as **Rust, C, C++, and many others**.
-- Rich language support already exists for Rust: you can use the Stylus SDK and CLI tool to start building today.
-- Stylus smart contracts benefit from **Arbitrum's EVM equivalence** thanks to a second, coequal WASM virtual machine.
-- Solidity contracts and Stylus contracts are fully interoperable. In Solidity, you can call a Rust program and vice versa.
-- Stylus contracts are significantly faster and have lower gas fees due to the superior efficiency of <a data-quicklook-from="wasm">WASM</a> programs.
-- Stylus makes it viable to consume RAM on the blockchain because it can greatly optimize memory use, enabling new use cases.
+- Stylus lets you write smart contracts in programming languages that compile to WASM, such as **Rust, C, C++, and many others** allowing you to tap into their ecosystem of libraries and tools. Rich language and tooling support already exists for Rust, try out the SDK and CLI with the [quickstart](https://docs.arbitrum.io/stylus/stylus-quickstart) today.
```

Comment:
```suggestion
- Stylus lets you write smart contracts in programming languages that compile to WASM, such as **Rust, C, C++, and many others** allowing you to tap into their ecosystem of libraries and tools. Rich language and tooling support already exists for Rust, try out the SDK and CLI with the [quickstart](https://docs.arbitrum.io/stylus/stylus-quickstart) today.
```
```suggestion
- Stylus lets you write smart contracts in programming languages that compile to WASM, such as **Rust, C, C++, and many others**, allowing you to tap into their ecosystem of libraries and tools. Rich language and tooling support already exists for Rust. You can try the SDK and CLI with the [quickstart](https://docs.arbitrum.io/stylus/stylus-quickstart).
```

Reply:
It sounds probably better with "soft" CTAs. OCL's docs tone is more declarative than promotional.